WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/ShippingLabelPackageDetailsViewModel.swift

```diff
@@ -104,12 +111,10 @@ final class ShippingLabelPackageDetailsViewModel: ObservableObject {
             guard let self = self else { return }
             self.products = products
             self.itemsRows = self.generateItemsRows()
-            self.setTotalWeight()
```

Comment:
Do you think it is safe to not calculate the total weight again when product list change? I know the chance that product list changes in this case is low - but maybe some extra safety is necessary?

Reply:
Thanks for catching this! I removed these from products/variations at one point and then meant to put them back, but somehow missed them — I agree it's better to have them there.